FLUTTER_VERSION

```diff
@@ -0,0 +1 @@
+flutter-3.27-candidate.0
```

Comment:
Can you use the pubspec.yaml version name for this?
You could use https://mikefarah.gitbook.io/yq/v2.x/read for this

Reply:
I can do it